layers/error_message/logging.h

```diff
@@ -219,8 +219,11 @@ class DebugReport {
         return FormatHandle(VkHandleInfo<T>::Typename(), HandleToUint64(handle));
     }
 
+    // Formats messages to be in the proper format, handles VUID logic, and any legacy issues
     bool LogMsg(VkFlags msg_flags, const LogObjectList &objects, const Location &loc, std::string_view vuid_text,
                 const char *format, va_list argptr);
+    // Core logging that interacts with the DebugCallbacks
+    bool DebugLogMsg(VkFlags msg_flags, const LogObjectList &objects, const char *message, const char *text_vuid) const;
```

Comment:
For consistency with `msg_flags`
```suggestion
    bool DebugLogMsg(VkFlags msg_flags, const LogObjectList &objects, const char *msg, const char *text_vuid) const;
```